assets/js/theme/common/product-details.js

```diff
@@ -463,6 +463,9 @@ export default class ProductDetails {
             // if the stock container is hidden, show
             viewModel.stock.$container.removeClass('u-hiddenVisually');
 
+            viewModel.stock.$input.text(data.stock);
+        } else {
+            viewModel.stock.$container.addClass('u-hiddenVisually');
             viewModel.stock.$input.text(data.stock);
```

Comment:
shouldn't this text be updating to be an empty string since the data.stock does not exist in this case?

Reply:
turns it out does not matter....this is working as is